Portal_Perfomance_Employes/Controllers/UserInfoController.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PortalPerfomanceEmployees.Data;
+using PortalPerfomanceEmployees.Models;
+using Microsoft.AspNetCore.Authorization;
+using System.Security.Claims;
+
+namespace PortalPerfomanceEmployees.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
+    public class UserInfoController : ControllerBase
+    {
+        private readonly AppDbContext _context;
+
+        public UserInfoController(AppDbContext context)
+        {
+            _context = context;
+        }
+
+        [Route("GetUserRole")]
+        [HttpGet]
+        public IActionResult GetUserRole()
+        {
+            return Ok(User.FindFirst(ClaimTypes.Role)?.Value);

```

Comment:
FindFirst throws an exception if no results are found, so I think we should try catch that if we want to use it. Otherwise we could use FirstOrDefault

Reply:
Corrected in a new commit